README.rst

```diff
@@ -265,6 +265,29 @@ before a function it will suppress the warning for that function.
        ...
    }
 
+
+Limitations
+-----------
+
+Lizard requires syntactically correct code.
+Upon processing input with incorrect or unknown syntax,
+Lizard's behavior is pretty much undefined (in good old C sense).
+It can ignore the code, throw an exception, crash, fail silently, or succeed.
```

Comment:
I am trying to think about how to reword this. The following is just brain dump.
```
In the case of invalid/partial/incorrect/unknown syntax:

- Guarantees (or tries its best to) no hard failures: exit, crash, exceptions.
- There is a chance of a combination of the following soft failures:
    - omission
    - misinterpretation
    - improper analysis / tally
    - success (the code in consideration is not relevant, e.g., global macros in C)

Developers of Lizard attempt to minimize the possibility of soft failures.
Hard failures are bugs in Lizard code,
while soft failures are trade-offs.
```